sdk/eventhub/eventhubs-checkpointstore-blob/samples/downlevelContainerClient.js

```diff
@@ -0,0 +1,42 @@
+/*
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  Licensed under the MIT Licence.
+
+  This sample demonstrates how to extend the Storage Blob Container Client
+  to downlevel the API version used when communicating with the service.
+
+  This may be useful if the environment you are targetting supports an
+  older version of Storage Blob than is officially supported by the
+  @azure/storage-blob SDK.
```

Comment:
Immediately after "....is officially supported by the @azure/storage-blob SDK." Please add "For example, if you are running Event Hubs on Azure Stack Hub, the Azure Stack Hub version 2002 supports up to version 2017-11-09 of the Azure Storage service. For more information of the Azure Storage service versions supported on Azure Stack, please refer to docs.microsoft.com/en-us/azure-stack/user/azure-stack-acs-differences."